pkgs/by-name/it/itstool/package.nix

```
@@ -1,7 +1,10 @@
-{ stdenv
-, lib
-, fetchurl
-, python3
+{
+  stdenv,
+  lib,
+  fetchurl,
+  buildPackages,
+  python3,
```

```suggestion
  python3Packages,
```

`python3Packages` is spliced so use it instead